api/config/config_prod.json

```diff
@@ -122,7 +122,7 @@
     "enableEventDateModifier": false,
     "enablePrivateDataprocWorker": false,
     "ccSupportWhenAdminLocking": true,
-    "enableRStudioGKEApp": false,
+    "enableRStudioGKEApp": true,
```

Comment:
We also need to set `showNewAnalysisTab` to true in UI environment.prod.ts (and also Stable)

Reply:
Also enable this flag in Stable.